### DISCUSSION TITLE
nvim-dap for Python via .vscode/launch.json

### DISCUSSION THREAD

**Discussion by hanweikung** [LazyVim/LazyVim#1707] [Q&A]
I am having issues while trying to configure nvim-dap to support debugging Python scripts using `.vscode/launch.json` configuration files.

I have already added the DAP core and Python plugins. I have also included the following line in my DAP core configuration:

```lua
require("dap.ext.vscode").load_launchjs(vim.fn.getcwd() .. "/.vscode/launch.json", {})
```

The snippet of the DAP core configuration is:

```lua
opts = {},
config = function(_, opts)
        -- setup dap config by VsCode launch.json file
    	require("dap.ext.vscode").load_launchjs(vim.fn.getcwd() .. "/.vscode/launch.json", {})
        local dap = require("dap")
        local dapui = require("dapui")
        dapui.setup(opts)
``` 

However, when I inserted a breakpoint into my Python script, initiated debug continue (`<leader>dc`), and selected `Python: Launch file`, it seems like the `.vscode/launch.json` file was not being loaded.

Would appreciate any suggestions on how to resolve this setup issue.

**Comment by xaviergmail** [comment]
Did you manage to make the vscode launch.json work?

**Reply by hanweikung** [reply]
Not in LazyVim. Instead, I found LunarVim to be an alternative solution.

https://medium.com/better-programming/lunarvim-debugging-testing-python-code-fa84f804c469
